### PR TITLE
PAL-390 resource service persistence

### PR DESCRIPTION
**Pull request by dev930018**

### CODE REVIEW

File: clients-mapreduce/src/test/java/uk/gov/gchq/palisade/clients/mapreduce/InputFormatUtilsTest.java [hunk 1]
```diff
@@ -46,40 +40,40 @@
...
-                .entrySet()
-                .stream()
-                .collect(Collectors.toList());
-        //Then - should equal original map
-        assertEquals(original, entries
-                .stream()
-                .collect(InputFormatUtils.listToMapCollector())
-        );
-    }
+//    @Test
```
[the remark this answers is not in the document]
Comment (dev930018) [reply clients-mapreduce/src/test/java/uk/gov/gchq/palisade/clients/mapreduce/InputFormatUtilsTest.java:43]: @ac74475 thoughts?

File: clients-mapreduce/src/test/java/uk/gov/gchq/palisade/clients/mapreduce/PalisadeInputSplitTest.java [hunk 2]
```diff
@@ -44,7 +43,7 @@ public void shouldntAcceptNegativeLength() throws IOException {
         Assert.fail("exception expected");
     }
 
-    private StubResource stubResource = new StubResource("test type", "test id", "test format");
+//    private StubResource stubResource = new StubResource("test type", "test id", "test format");
```
Comment (ac74475) [inline clients-mapreduce/src/test/java/uk/gov/gchq/palisade/clients/mapreduce/PalisadeInputSplitTest.java:46]: Why are the lines around StubResource commented out? either remove the lines completely or add comments to the code to explain

File: clients-mapreduce/src/test/java/uk/gov/gchq/palisade/clients/mapreduce/InputFormatUtilsTest.java [hunk 3]
```diff
@@ -46,41 +40,6 @@
 //        reqResponse.originalRequestId(new RequestId().id("test"));
```
Comment (developer01189998819991197253) [inline clients-mapreduce/src/test/java/uk/gov/gchq/palisade/clients/mapreduce/InputFormatUtilsTest.java:40]: whats the status of this @BeforeClass and setup? if commented out can it be removed?

Comment (dev930018) [reply clients-mapreduce/src/test/java/uk/gov/gchq/palisade/clients/mapreduce/InputFormatUtilsTest.java:40]: They are part of the set-up for other commented-out tests

**Pull request merged**